rules/exec_properties.bzl

```diff
@@ -0,0 +1,134 @@
+# Copyright 2017 The Bazel Authors. All rights reserved.
```

Comment:
lets move these to a subdirectory under rules
could be 
rules/rbe or something like that

Reply:
I see that there is a directory rules/rbe_repo/ does it make sense to put it there or would you rather I create a new directory rules/rbe/ ?

Reply:
Following our offline conversation, I'm putting this in rules/experimental/rbe/ for now.